tests/upgradePMM_test.js

```diff
@@ -31,15 +38,22 @@ Before(async ({ I }) => {
 });
 
 Scenario(
-  'PMM-T289 Verify Whats New link is presented on Update Widget @pre-upgrade @pmm-upgrade @not-ui-pipeline',
+  'Add AMI Instance ID @ami-upgrade',
+  async ({ I, amiInstanceAPI }) => {
+    await amiInstanceAPI.verifyAmazonInstanceId(process.env.AMI_INSTANCE_ID);
+  },
+);
+
+Scenario(
+  'PMM-T289 Verify Whats New link is presented on Update Widget @ami-upgrade @pre-upgrade @pmm-upgrade @not-ui-pipeline',
```

Comment:
could you please remove `@not-ui-pipeline` tags ?

Reply:
yes, forgot will update.